### DISCUSSION TITLE
Examples of projects with AdMob implementation

### DISCUSSION THREAD

**Discussion by OziOcb** [quasarframework/quasar#10236] [CLI - Capacitor / Cordova modes]
Hi.
Like in the title, does anyone know any good example of projects on GitHub/GitLab that have AdMob implemented?
Thank you

**Comment by ontwikkelfabriek** [comment]
@OziOcb 

There's Qodo (pretty complex project)
https://github.com/vycoder/qodo

**Reply by OziOcb** [reply]
Thank you! I'll have a look and hopefully I'll learn from it :)

**Reply by capslockoff** [reply]
Is there any with cordova?
